lib/cm_admin/view_helpers/page_info_helper.rb

```diff
@@ -5,6 +5,42 @@ def page_title
         @action.title || @model.title || "#{@model.ar_model.name} | #{@action.name&.titleize} | Admin"
       end
 
+      def action_title
+        if @model.current_action.page_title
+          title = @model.current_action.page_title
+        else
+          title = "#{@model.name}"
+          case action_name
+          when 'index'
+            title + " list record"
+          when 'show'
+            title + " detail"
+          when 'new'
+            title + " create record"
+          when 'edit'
+            title + " edit record"
+          end
+        end
+      end
+
+      def action_description
+        if @model.current_action.page_title
+          title = @model.current_action.page_title
```

Comment:
@anbublacky - This method should be for `page_description` right?
```suggestion
        if @model.current_action.page_description
          title = @model.current_action.page_description
```